app/mailers/user_mailer.rb

```diff
@@ -322,10 +322,12 @@ def in_person_ready_to_verify_reminder(enrollment:)
       code: enrollment.enrollment_code,
     ).image_data
 
+    @is_enhanced_ipp = enrollment.enhanced_ipp?
     with_user_locale(user) do
       @presenter = Idv::InPerson::ReadyToVerifyPresenter.new(
         enrollment: enrollment,
         barcode_image_url: attachments['barcode.png'].url,
+        is_enhanced_ipp: @is_enhanced_ipp,
```

Comment:
Viewed in_person_ready_to_verify_reminder email template for EIPP in all languages and it looks as I'd expect. The IPP reminder is not affected. I want to run through to test with the flow on Monday but do not expect any problems.